### DISCUSSION TITLE
Monitor stopped being detected

### DISCUSSION THREAD

**Discussion by Mohs9n** [hyprwm/Hyprland#2216] [Q&A]
I installed the [ly](https://github.com/fairyglade/ly) display manager and used it to open Hyprland and it worked but the external monitor (I'm using a laptop) didn't show up so i ran `hyprctl monitors` and `wlr-randr` didn't detect it either, i disabled and uninstalled ly and rebooted but it still doesn't show up, I tested it on X with qtile and It's working fine, is there anything i can do to figure out the problem? 

Edit: I installed Hyprland a couple of days ago and the monitor was working until now

**Comment by x55xaa** [comment]
I'm having the same problem. Did you find a solution?

**Reply by Mohs9n** [reply]
I don't really remember but I don't think it was a  specific display manager problem, I think I distro hopped or did a reinstall of arch so sadly I don't have a solution.
